*** common.inc.orig	2008-07-15 19:30:30.000000000 +0400
--- common.inc	2008-07-15 21:17:20.000000000 +0400
***************
*** 1149,1154 ****
--- 1149,1155 ----
   *   A translated date string in the requested format.
   */
  function format_date($timestamp, $type = 'medium', $format = '', $timezone = NULL, $langcode = NULL) {
+   if(function_exists('dst_format_date') ) return dst_format_date($timestamp, $type, $format, $timezone, $langcode);
    if (!isset($timezone)) {
      global $user;
      if (variable_get('configurable_timezones', 1) && $user->uid && strlen($user->timezone)) {
